dashboard-api/docker-compose-test.yaml

```diff
@@ -0,0 +1,41 @@
+services:
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile.test
+    environment:
+      STUDIESMANAGER_DBURI: postgresql://root@cockroachdb:26257/vlab?sslmode=disable
```

Comment:
For consistency reasons, shouldn't we use the same env var declarations everywhere? Currently, in `system` we use https://github.com/vlab-research/vlab/pull/75/files#diff-fc992c9a9d4e6e1728cac26d958fcbe0cde48518b65c36fbd6740aa47d85b8a3R30-R33.